src/ui/wxWidgets/PWFiltersDateDlg.cpp

```diff
@@ -589,7 +589,7 @@ bool pwFiltersDateDlg::IsValid(bool showMessage) const {
     // All dates, except History must be located in the history
     if ((fdate1 > wxDateTime::Now()) || (isRuleSelected(idx, PWSMatch::MR_BETWEEN) && (fdate2 > wxDateTime::Now()))) {
       if (showMessage) {
-        wxMessageBox(_("Please correct date values."), _("A future date is not valid for this field"), wxOK|wxICON_ERROR);
+        wxMessageBox(_("Correct date values."), _("A future date is not valid for this field"), wxOK|wxICON_ERROR);
```

Comment:
I would keep the old messages in this file

Reply:
Agree.